include/flamegpu/model/LayerDescription.h

```diff
@@ -319,8 +314,16 @@ void LayerDescription::addAgentFunction(AgentFunction /*af*/) {
         "in LayerDescription::addAgentFunction().");
 }
 
-// Not included in ifdef SWIG intentionally - needs to be available for DependencyGraph to call
+// Can't be moved to cpp file because of inline hint
```

Comment:
So remove the inline hint??

Reply:
Assumed whoever wrote it put it there for good reason but I can remove it haha